rest-service/manager_rest/rest/endpoint_mapper.py

```diff
@@ -108,7 +109,6 @@ def setup_resources(api):
         'SummarizeExecutions': 'summary/executions',
         'SummarizeBlueprints': 'summary/blueprints',
         'SummarizeExecutionSchedules': 'summary/execution_schedules',
-        'UserTokens': 'user-tokens/<string:user_id>',
```

Comment:
so of course this is not back-compatible, as you are aware; it'll also need rest-client updates

Reply:
Yup, it's on the radar.